ant_world/parameters.h

```diff
@@ -68,7 +75,7 @@ namespace Parameters
     constexpr unsigned int displayRenderHeight = intermediateSnapshotHeight * displayScale;
 
     // Ant parameters
-    // How fast does the ant move?
-    constexpr float antTurnSpeed = 4.0f;
-    constexpr float antMoveSpeed = 0.05f;
+    // How fast does the ant move each timestep?
+    constexpr degree_t antTurnSpeed = 4.0_deg;
+    constexpr meter_t antMoveSpeed = 0.05_m;
```

Comment:
Just a minor nit: I'm not sure "speed" is the right word here (I thought you meant real velocity for a mo). How about ``antTurnStepSize`` and ``antStepSize`` (or something less clunky)?

Reply:
Hah yeah - the dubious physics of the antworld is exposed :grimacing: